components/squad/commons/squad_row.lua

```diff
@@ -121,6 +121,7 @@ function SquadRow:name(args)
 	return self
 end
 
+---Position and Role shouldn't both be used, only one
```

Comment:
Is this accurate? I thought some wikis use position to have ingame positions listed, whereas role is for the usual stuff and could be e.g. inactive while still having an ingame role?

Reply:
Both Position and Role functions shouldn't be used for a single row, as the position one also shows the role
I'll fix the comment